test/e2e/artifacts_test.go

```diff
@@ -392,6 +408,8 @@ func (s *ArtifactsSuite) TestArtifactGC() {
 			DeleteWorkflow().
 			WaitForWorkflowDeletion()
 
+		when.Then().ExpectWorkflowDeleted()
+
```

Comment:
```suggestion
			Then().
			ExpectWorkflowDeleted()

```

can this be put into the chain?